src/components/album/Album.js

```diff
@@ -53,23 +80,40 @@ function Album({ album, wishListId }) {
         <AddToWishlistButton album={album} wishListId={wishListId} />
       </ButtonGroup>
       {/* Dialog */}
-      <Dialog open={openDialog} onClose={handleCloseDialog}>
+      <Dialog
+        open={openDialog}
+        onClose={handleCloseDialog}
+        fullWidth={fullWidth}
+        maxWidth={maxWidth}
+      >
         <DialogActions>
           <Button onClick={handleCloseDialog} color="primary">
             X
           </Button>
         </DialogActions>
         <DialogContent>
-          <DialogContentText>
+          <DialogContentText  style={{
+          
+          display: 'flex',
+          flexDirection: 'column',
+          justifyContent: 'center',
+          alignItems: 'center',
```

Comment:
<img width="679" alt="Screenshot 2023-11-13 at 10 21 37 AM" src="https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/4dd0a695-a121-41f1-a585-7dd57a402aec">

I'm not super sure about this style of having _everything_ centered in the modal; it seems to leave a lot of white space 🤔